core/sds-aspect-model-validator/src/main/java/io/openmanufacturing/sds/aspectmodel/validation/services/AspectModelValidator.java

```diff
@@ -167,6 +171,9 @@ public List<Violation> validateModel( final Try<VersionedModel> versionedModel )
          // To catch false positives, also try to load the model
          final Try<List<ModelElement>> modelElements = AspectModelLoader.getElements( model );
          if ( modelElements.isFailure() && !(modelElements.getCause() instanceof InvalidRootElementCountException) ) {
+            final ByteArrayOutputStream out = new ByteArrayOutputStream();
```

Comment:
Will this remain or was it meant only for debugging?

Reply:
Yes, this should be removed here.